pkg/deploy/generator/scripts/gatewayVMSS.sh

```diff
@@ -67,6 +72,19 @@ include /etc/logrotate.d
     create 0600 root utmp
     rotate 1
 }
+
+# Maximum log directory size is 100G with this configuration
+# Setting limit to 100G to allow space for other logging services
+# copytruncate is a critical option used to prevent logs from being shipped twice
+${gateway_logdir} {
+    size 20G
+    rotate 5
+    create 0600 root root
+    copytruncate
+    nooldir
```

Comment:
Typo?
```suggestion
    noolddir
```

Reply:
Yes, thanks. I should try to copy/paste config options more.